packages/admin-sdk/src/_internals/utils.ts

```diff
@@ -1,4 +1,8 @@
+/**
+ *
+ */
```

Comment:
Is there a particular reason for this to exist? Has this any benefit?

Reply:
Good catch! Thank you 😊